packages/devtools/lib/src/memory/memory.dart

```diff
@@ -410,6 +410,23 @@ class MemoryScreen extends Screen with SetStateMixin {
     heapAutoCompletePopup.hide();
   }
 
+  void _selectInstanceByObjectRef(String objectRefToFind) {
+    removeInstanceView();
+
+    // There's an instances table up.
+    final Table<Object> instanceTable = tableStack.last;
+    final List<InboundsTreeNode> nodes = instanceTable.model.data;
+
+    int row = 0;
+    for (var node in nodes) {
+      if (node.instance?.objectRef == objectRefToFind) {
+        instanceTable.selectByIndex(row, scrollBehavior: 'auto');
+        return;
+      }
+      row++;
```

Comment:
instead of storing this row var, would `nodes.indexOf(node)` accomplish the same end?

Reply:
Done.